src/commands/replicaof.json

```diff
@@ -14,12 +14,40 @@
         ],
         "arguments": [
             {
-                "name": "host",
-                "type": "string"
-            },
-            {
-                "name": "port",
-                "type": "integer"
```

Comment:
I'll repost that I still think this is the wrong solution. I think we should just have changed the type of port to string.

Reply:
it's not just correctness, it's also for documentation.
e.g. it'll make the completion tips in redis-cli clearer.